README.md

```diff
@@ -784,6 +784,41 @@ let app =
     ]
 ```
 
+#### Gotchas
+
+If your route is not returning static html, then you should wrap your function with a warbler.
+
+```fsharp
+// unit -> string
+let model() =
+    System.DateTime.Now.ToString()
+
+// HtmlNode
+let staticView =
+    p [] (model() |> rawText)
+
+// unit -> HtmlNode
+let dynamicView() =
+    p [] (model() |> rawText)
+
+let webApp = 
+    choose [
+        GET >=>
+            choose [
+                route "/static"      >=> (staticView |> renderHtml)
+                route "/dynamic"     >=> warbler (fun _ -> dynamicView() |> renderHtml)
+            ]
+    ]
+```
+
+Functions in F# are eagerly evaluated and in the case of razor/htmlEngine a warbler will help to evaluate the function every time.
+
+```fsharp
+// ('a -> 'a -> 'b) -> 'a -> 'b
+let warbler f a =
+    f a a
+```
+
```

Comment:
This is a good example. I was thinking perhaps we should move the section `Gotchas` beneath the list of default http handlers or perhaps move it under `warbler` in the http handlers, because it applies really to all handlers and not just the html or razor views. The same rules apply for `text`, `json`, `xml`, `negotiate`, etc.
What do you think?